holopy/scattering/tests/common.py

```diff
@@ -26,7 +26,7 @@
 pixel_scale = [.1151e-6, .1151e-6]
 index = 1.33
 
-xschema = update_metadata(detector_grid(shape = 128, spacing=pixel_scale), illum_wavelen=wavelen, medium_index=index, illum_polarization=xpolarization)
+xschema = update_metadata(detector_grid(shape=64, spacing=pixel_scale), illum_wavelen=wavelen, medium_index=index, illum_polarization=xpolarization)
```

Comment:
Is this the cause of the failing tests?

Reply:
Yes, but reverting this change breaks the raw fields tests in test_lens. I'll just make them not use the same detector for now until I sort this out.